atbuild/src/main.swift

```diff
@@ -27,7 +27,7 @@ enum Options: String {
 
 let defaultPackageFile = "build.atpkg"
 
-func loadPackageFile() -> Package {
+func loadPackageFile(task: String?) -> Package {
```

Comment:
This argument needs a label. Without it, it looks like should be loading the package from the first parameter.

`func loadPackageFile(focusOnTask task: String) -> Package`

But really, this function seems like it's just doing way too much now and should be refactored into getting the package filename, getting the overlays, and then just having the creation of the `Package` be outside.